### DISCUSSION TITLE
How to stop your vercel bot from comment in other github code?

### DISCUSSION THREAD

**Discussion by skykingit** [vercel/vercel#6950] [Help] [answered]
I just login vercel with my github account
but your bot always comment in other github code?
how to stop it..............................

**Comment by PaulRBerg** [comment, marked as answer]
You can do this by adding a `vercel.json` file with the following contents:

```json
{
  "github": {
    "silent": true
  }
}
```

See the docs: [How do I prevent the Vercel for GitHub integration comments?](https://vercel.com/guides/how-to-prevent-vercel-github-comments).
